File: hw/ramos/u1/HW11ChickenCoops/Python/Poop.py
class Poop:
    def __init__(self, amount: int):
        self.__amount = amount  

    def __str__(self):
        return f"Poop{{amount={self.__amount}}}"

    # Getter
    def get_amount(self) -> int:
        return self.__amount

    # Setter
    def set_amount(self, amount: int):
        self.__amount = amount

File: ws/collaguazo/u1/CodingClasses/model/poop.java
package ec.edu.ec.chickenfarmsimulator.model;

/**
 *
 * @author Pablo Collaguazo
 */
public class Poop {
    private int amount;

    @Override
    public String toString() {
        return "Poop{" + "amount=" + amount + '}';
    }

    public Poop(int amount){
        this.amount = amount;
    }
    public int getAmount() {
        return amount;
    }

    public void setAmount(int amount) {
        this.amount = amount;
    }
    
}
